src/index.js

```diff
@@ -0,0 +1,28 @@
+import apis from '../apis.json';
```

Comment:
It is included in the browser bundle, that's how the clients are generated. I'm not sure what the problem is here, could you elaborate?